awx/ui_next/src/components/DataListToolbar/DataListToolbar.jsx

```diff
@@ -7,38 +7,23 @@ import styled from 'styled-components';
 import { SearchIcon } from '@patternfly/react-icons';
 import {
   DataToolbar,
-  DataToolbarContent,
-  DataToolbarGroup,
-  DataToolbarToggleGroup,
+  DataToolbarContent as _DataToolbarContent,
+  DataToolbarGroup as _DataToolbarGroup,
   DataToolbarItem,
+  DataToolbarToggleGroup,
 } from '@patternfly/react-core/dist/umd/experimental';
 import ExpandCollapse from '../ExpandCollapse';
 import Search from '../Search';
 import Sort from '../Sort';
 
 import { SearchColumns, SortColumns, QSConfig } from '@types';
 
-const AdditionalControlsWrapper = styled.div`
-  display: flex;
-  flex-grow: 1;
-  justify-content: flex-end;
-  align-items: center;
-
-  & > :not(:first-child) {
-    margin-left: 20px;
-  }
-`;
-
-const AdditionalControlsDataToolbarGroup = styled(DataToolbarGroup)`
-  margin-left: auto;
-  margin-right: 0 !important;
+const DataToolbarContent = styled(_DataToolbarContent)`
```

Comment:
do we want to adjust these values?

Reply:
Those styles vertically align the first item of the DataListToolbar with the first item of the DataListRow. PatternFly core wraps the DataListToolbar in 24px of padding by default, so whenever PF React updates their DataListToolbar styles we could remove these values. I'm fine with removing them now if we prefer to wait for the upstream fix instead of adjusting these values.

<img width="275" alt="Screen Shot 2020-02-19 at 4 27 47 PM" src="https://user-images.githubusercontent.com/15881645/74878421-45582c80-5335-11ea-9eab-9352977fd469.png">